src/surfaces/actionBar/actionBar.jsx

```diff
@@ -229,12 +236,12 @@ class ActionBar extends React.PureComponent {
                                                 </DropdownButton>
                                             )}
 
-                                            {list && _.isArray(list) && (
+                                            {list && isArray(list) && (
```

Comment:
`list && isArray(list)` is probably redundant.  Could probably just be `isArray(list)`

Reply:
Fixed.

Reply:
Looks good.  Resolving.